test/conformance/exp_command_buffer/fixtures.h

```diff
@@ -139,19 +102,65 @@ struct urCommandBufferExpExecutionTest : uur::urKernelExecutionTest {
     }
 
     ur_exp_command_buffer_handle_t cmd_buf_handle = nullptr;
-    ur_bool_t updatable_command_buffer_support = false;
-    ur_platform_backend_t backend{};
 };
 
-struct urUpdatableCommandBufferExpExecutionTest
-    : urCommandBufferExpExecutionTest {
+struct urUpdatableCommandBufferExpTest : uur::urQueueTest {
     void SetUp() override {
-        UUR_RETURN_ON_FATAL_FAILURE(urCommandBufferExpExecutionTest ::SetUp());
+        UUR_RETURN_ON_FATAL_FAILURE(uur::urQueueTest::SetUp());
+
+        UUR_RETURN_ON_FATAL_FAILURE(checkCommandBufferSupport(device));
+        UUR_RETURN_ON_FATAL_FAILURE(checkCommandBufferUpdateSupport(device));
 
-        if (!updatable_command_buffer_support) {
-            GTEST_SKIP() << "Updating EXP command-buffers is not supported.";
+        // Create a command-buffer with update enabled.
+        ur_exp_command_buffer_desc_t desc{
+            UR_STRUCTURE_TYPE_EXP_COMMAND_BUFFER_DESC, nullptr, true};
+
+        ASSERT_SUCCESS(urCommandBufferCreateExp(context, device, &desc,
+                                                &updatable_cmd_buf_handle));
+        ASSERT_NE(updatable_cmd_buf_handle, nullptr);
+
+        // Currently there are synchronization issue with immediate submission when used for command buffers.
+        // So, create queue with batched submission for this test suite if the backend is Level Zero.
+        if (backend == UR_PLATFORM_BACKEND_LEVEL_ZERO) {
```

Comment:
Noting https://github.com/oneapi-src/unified-runtime/pull/2046 that I don't think we need this